.gitignore

```
@@ -1 +1,4 @@
 book
+.direnv
+src/assets/cli/cli_cheatsheet.pdf
+src/assets/editor/editor_cheatsheet.pdf
```

Borde lägga till bilden också, den borde också plockas bort från repot.
```suggestion
src/assets/cli/cli_cheatsheet.pdf
src/assets/editor/editor_cheatsheet.pdf
src/assets/editor/editor_cheatsheet.png
```

---

Hade hellre haft en hel mapp med genererade assets än att ignorerar specifika filer

---

Done